src/main/java/seedu/address/model/person/TelegramHandle.java

```diff
@@ -11,13 +11,14 @@ public class TelegramHandle {
 
     public static final TelegramHandle NULL_TELEGRAM_HANDLE = null;
     public static final String MESSAGE_CONSTRAINTS = "Telegram Handle must "
-            + "use a-z, 0-9 and underscores. Minimum length is 5 characters";
+            + "use a-z, 0-9 and underscores. Minimum length is 5 characters. It must start with a letter and cannot "
+            + "end with an underscore";
 
     /*
      * The telegram handle uses a-z, 0-9 and underscores.
      * Minimum length is 5 characters.
      */
```

Comment:
Did you forget to update the comment explaining the regex to be similar to `MESSAGE_CONSTRAINTS`?

Reply:
updated thanks